examples/analytics-vendors.amp.html.rej

```diff
@@ -0,0 +1,30 @@
+--- examples/analytics-vendors.amp.html
```

Comment:
You probably don't want to include this.

Reply:
fixed